package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.3",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10 <12"
+    "node": ">=10 <13"
```

Comment:
AFAIK the recommended Node version is 12. Why do we loosen it here to 13?

Reply:
By accident.